src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs

```diff
@@ -398,6 +395,13 @@ private static string ConvertIcuTimeFormatString(ReadOnlySpan<char> icuFormatStr
                         }
                         break;
 
+                    default:
+                        // Treat non-ASCII characters as literals
```

Comment:
Why you think we need to do that? this can cause a problem in the future if a new translation is introduced and we are blindly copying it to the translated pattern.

Reply:
Only ASCII letters are used as pattern symbols in the ICU date time format. It should be safe to copy all other symbols as they are just literal texts.

Reply:
Take `B` as an example. if we have this suggested code before, the pattern will end up having `B` and will be displayed as `B` in the formatted time which will be very ugly.

Reply:
Take a closer look. Unrecognized ASCII letters are removed from the pattern.

Reply:
We crossed our replies. please look at my previous reply.

Reply:
https://www.unicode.org/reports/tr35/tr35-dates.html#date-format-patterns

> Literal text, which is output as-is when formatting, and must closely match when parsing. Literal text can include:
> - Any characters other than A..Z and a..z, including spaces and punctuation.
> - Any text between single vertical quotes ('xxxx'), which may include A..Z and a..z as literal text.
> - Two adjacent single vertical quotes (''), which represent a literal single quote, either inside or outside quoted text.

Actually I need to fix the double adjacent single vertical quotes pattern, which is not covered in the original implementation as well.

Reply:
Ok, you are right.

Reply:
Literal single quote and literal backslash conversion fixed.